docs/api.md

```diff
@@ -0,0 +1,25 @@
+# Tock API
+
+[:arrow_left: Back to Tock
+Documentation](https://github.com/18F/tock/tree/master/docs)
+
+Tock has an API! You may issue GET requests to various [endpoints](https://github.com/18F/tock/tree/master/api-docs) via the /api/ path with results returned as JSON objects. We use Django REST framework's TokenAuthentication library which requires all requests to include a token value in your request header using the following format (a cURL command line based request is used for this example for getting project data from our Tock deployment):
```

Comment:
I'd strike this first "Tock has an API!" sentence. I know that from the header, and I'm reading how to use it.